web/single-object.html

```diff
@@ -18,6 +18,12 @@
             <div class="row">
               <div class="col text-center mb-4 mt-3 mt-lg-0">
                 <h6>Real-time Distribution of <span class="text-muted file-title-distribution">{{.Data.Key}}</span> on Tardigrade</h6>
+                <div id="smallFileMessage" class="row flex-nowrap text-left">
+                  <svg id="warningIcon" width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
There shouldn't be a warning icon. I guess, just text would be fine. It's completely normal behavior of the satellite.

Reply:
alright